diff --git a/dist/features/module-loading.js b/dist/features/module-loading.js
index 82655b699395e5a556d34447b8ddbceb27a6fd65..206037994c7350181b409ee4d0a1b3ddc4c72924 100644
--- a/dist/features/module-loading.js
+++ b/dist/features/module-loading.js
@@ -115,7 +115,9 @@ var moduleLoading = mixin({
         tableInitial,
         multithreaded,
       } = this.options = options;
+      // MIKE CHANGE START
       const res = await source;
+      // MIKE CHANGE END
       const suffix = (res[Symbol.toStringTag] === 'Response') ? 'Streaming' : '';
       const w = WebAssembly;
       const f = w['compile' + suffix];
@@ -145,7 +147,11 @@ var moduleLoading = mixin({
         shared: multithreaded,
       });
       this.initialTableLength = tableInitial;
-      return new w.Instance(executable, exports);
+      // MIKE CHANGE START
+      //return new w.Instance(executable, exports);
+      const isResp = res[Symbol.toStringTag] === 'Response';
+      return w['instantiate' + suffix](isResp ? fetch(res.url) : res, exports).then(w => w.instance); // gross, but don't wanna refactor
+      // MIKE CHANGE END
     },
     loadModule(source, options) {
       return this.initPromise = (async () => {
