diff --git a/dist/features/module-loading.js b/dist/features/module-loading.js
index 82655b699395e5a556d34447b8ddbceb27a6fd65..69287312a98c251435a49f7b69d2b99b2bce44e2 100644
--- a/dist/features/module-loading.js
+++ b/dist/features/module-loading.js
@@ -115,7 +115,10 @@ var moduleLoading = mixin({
         tableInitial,
         multithreaded,
       } = this.options = options;
-      const res = await source;
+      // MIKE CHANGE START
+      const res = await source;
+      console.log("mike source", source);
+      // MIKE CHANGE END
       const suffix = (res[Symbol.toStringTag] === 'Response') ? 'Streaming' : '';
       const w = WebAssembly;
       const f = w['compile' + suffix];
@@ -145,7 +148,10 @@ var moduleLoading = mixin({
         shared: multithreaded,
       });
       this.initialTableLength = tableInitial;
-      return new w.Instance(executable, exports);
+      // MIKE CHANGE START
+      //return new w.Instance(executable, exports);
+      return WebAssembly.instantiateStreaming(fetch(res.url), exports).then(w => w.instance); // gross, but don't wanna refactor
+      // MIKE CHANGE END
     },
     loadModule(source, options) {
       return this.initPromise = (async () => {
